ladybug_geometry/geometry3d/face.py

```diff
@@ -1201,22 +1205,35 @@ def extract_rectangle(self, tolerance):
         """Extract top and bottom line segments of a rectangle within this Face.
 
         This method will only return geometry if:
-            1. There are no holes in the face.
-            2. The face is not parallel to the World XY plane.
-            3. There are two parallel edges to this face, which are either
-                oriented horizontally or vertically.
-            4. There must be enough overlap between these edges for a rectangle
-                to be drawn between them.
+
+        1) There are no holes in the face.
+
+        2) The face is not parallel to the World XY plane.
+
+        3)
+            There are two parallel edges to this face, which are either
+            oriented horizontally or vertically.
+
+        4)
+            There must be enough overlap between these edges for a rectangle
+            to be drawn between them.
+
         If this Face does not satisfy this criteria, None will be returned.
 
         Args:
             tolerance: The maximum difference between point values for them to be
                 considered a part of a rectangle.
 
         Returns:
-            bottom_edge: A LineSegment3D representing the bottom of the rectangle.
-            top_edge: A LineSegment3D representing the top of the rectangle.
-            other_faces: A list of Face3D objects for the parts of this face not
+            A list with three elements
+
+            - bottom_edge: A LineSegment3D representing the bottom of the rectangle.
+
+            - top_edge: A LineSegment3D representing the top of the rectangle.
+
+            -
+                other_faces:
```

Comment:
Again, there has to be a better solution to this case.  I guess another option might be to use an extra indent for the first two items (`bottom_edge` and `top_edge`) just to make things consistent.

Reply:
Fixed, a tab space did it!